packages/minimongo/common.js

```diff
@@ -87,8 +87,41 @@ export const ELEMENT_OPERATORS = {
     // should *not* include it itself.
     dontIncludeLeafArrays: true,
     compileElementSelector(operand) {
-      if (typeof operand !== 'number') {
-        throw Error('$type needs a number');
+      if (typeof operand === 'string') {
+        const operandAliasMap = {
+          'double': 1,
+          'string': 2,
+          'object': 3,
+          'array': 4,
+          'binData': 5,
+          'undefined': 6,
+          'objectId': 7,
+          'bool': 8,
+          'date': 9,
+          'null': 10,
+          'regex': 11,
+          'dbPointer': 12,
+          'javascript': 13,
+          'symbol': 14,
+          'javascriptWithScope': 15,
+          'int': 16,
+          'timestamp': 17,
+          'long': 18,
+          'decimal': 19,
+          'minKey': -1,
+          'maxKey': 127,
```

Comment:
Is there any way we could get this information from some npm package, or from the `mongo` package, somehow?

Reply:
Did a quick search and couldn't find anything (the mongodb npm driver at least doesn't contain it). Also, the `mongo` package has `minimongo` as a dependency, not the other way around.

Reply:
OK, I can't find anything useful either!